public/lib/css/index.css

```diff
@@ -901,15 +901,92 @@ header a:hover {
   margin-bottom: 0
 }
 
+.herofive-block .herofive-block-content .portfolio .portfolio-content#feature-project-1{
+  height: 300px;
+  border-radius: 0.25em;
+  width:230px;
+  background:url(http://www.pageplusnc.com/v/vspfiles/photos/iPhone-5-1.jpg);
+  background-size:cover;
+  overflow:hidden;
+  margin: 0px;
+  padding:0px;
+}
+
+.herofive-block .herofive-block-content .portfolio .portfolio-content#feature-project-2{
+  height: 300px;
+  border-radius: 0.25em;
+  width:230px;
+  background:url(http://www.pageplusnc.com/v/vspfiles/photos/iPhone-5-1.jpg);
+  background-size:cover;
+  overflow:hidden;
+  margin: 0px;
+  padding:0px;
+}
+
+.herofive-block .herofive-block-content .portfolio .portfolio-content#feature-project-3{
+  height: 300px;
+  border-radius: 0.25em;
+  width:230px;
+  background:url(http://www.pageplusnc.com/v/vspfiles/photos/iPhone-5-1.jpg);
+  background-size:cover;
+  overflow:hidden;
+  margin: 0px;
+  padding:0px;
+}
+
+.herofive-block .herofive-block-content .portfolio .portfolio-content#feature-project-4{
+  height: 300px;
+  border-radius: 0.25em;
+  width:230px;
+  background:url(http://www.pageplusnc.com/v/vspfiles/photos/iPhone-5-1.jpg);
+  background-size:cover;
+  overflow:hidden;
+  margin: 0px;
+  padding:0px;
+}
+
+.herofive-block .herofive-block-content .portfolio .portfolio-content .flyout{
+  background:#3D525C;
+  text-align:center;
+  opacity:0;
+  -webkit-transition: opacity .25s ease;
+  -moz-transition: opacity .25s ease;
+  height: 300px;
+  width:230px;
+  margin: 0px;
+  border-radius: 0.25em;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  text-align: center;
+}
+
+.herofive-block .herofive-block-content .portfolio .portfolio-content .flyout p{
+  color:white;
+}
+
+.herofive-block .herofive-block-content .portfolio .portfolio-content .flyout .button{
+  margin:auto;
+  margin-top:60px;
+  width:150px;
+}
+
+.herofive-block .herofive-block-content .portfolio .portfolio-content .flyout:hover{
+  opacity:0.9;
+}
+
 @media (min-width:641px) {
   .herofive-block-content {
     padding-top: 2.5em;
     padding-bottom: 2.5em;
   }
   .portfolio {
+    /*margin:0 20px;*/
     padding-top: 3em;
     padding-bottom: 3em;
+    width: 230px;
     float: left;
+    margin-right: 1.6949152542%;
```

Comment:
This is present in our team section on the index page as well, this is pretty much from the compiled css that kevin gave us off from the yeoman-ed project. At first i wanted to use a manual margin (hence a margin declaration not stated above), but it's safe to use this as well.